### PR TITLE
[WNMGDS-2797] Update CSS var snapshots

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

Updating the CSS-var snapshots so I can annotate and double-check the changes we're making.

### CODE REVIEW

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 1]
```diff
@@ -22,13 +25,10 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
 --alert__font-color: #262626;
 --alert__icon-size: 1.5rem;
 --alert__padding: 16px;
---animation-ease-in-out-expo: cubic-bezier(1, 0, 0, 1);
---animation-speed-1: 0.25s;
---animation-speed-2: 0.3s;
---animation-speed-3: 0.5s;
---animation-speed-4: 0.8s;
---animation-speed-base: 1;
---article-max-width: 600px;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:31]: We don't use this variable and removed it

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 2]
```diff
@@ -297,22 +300,28 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
...
 --filter-chip__color: #262626;
+--font-family-body: 'Open Sans', 'Inter', 'Roboto', 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', 'Arial', 'sans-serif';
+--font-family-button: 'Open Sans', 'Inter', 'Roboto', 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', 'Arial', 'sans-serif';
+--font-family-heading: 'Open Sans', 'Inter', 'Roboto', 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', 'Arial', 'sans-serif';
+--font-family-link: 'Open Sans', 'Inter', 'Roboto', 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', 'Arial', 'sans-serif';
 --font-line-height-base: 1.5;
---font-line-height-heading: 1.3;
---font-line-height-lead: 1.7;
+--font-line-height-heading: 1.2999999523162842;
+--font-line-height-lead: 1.7000000476837158;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:324]: Some floating point funny business moving between Figma and JavaScript 🤷‍♂️

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 3]
```diff
@@ -326,28 +335,41 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
...
 --inline-error__color: #e31c3d;
---label__font-family: inherit;
---label__font-size: inherit;
+--label__font-family: 'Open Sans', 'Inter', 'Roboto', 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', 'Arial', 'sans-serif';
+--label__font-size: 1rem;
 --label__font-weight: 700;
---label__letter-spacing: inherit;
+--label__letter-spacing: 0px;
 --label__text-transform: inherit;
---lead-max-width: 77rem;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:350]: Also removed this

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 4]
```diff
@@ -367,7 +389,6 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
 --media-width-sm: 544px;
 --media-width-xl: 1280px;
 --media-width-xs: 0px;
---nav-width: 951px;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:370]: And removed this

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 5]
```diff
@@ -433,20 +446,22 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
...
 --text-input__border-radius: 3px;
+--text-input__border-width--disabled: 2px;
 --text-input__border-width--error: 3px;
 --text-input__border-width: 2px;
 --text-input__color: #262626;
 --text-input__divider-color: #a6a6a6;
---text-input__line-height: 1.3;
+--text-input__line-height: 1.2999999523162842;
 --text-input__padding: 8px;
---text-max-width: 53rem;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:449]: And this

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 6]
```diff
@@ -1862,32 +1792,44 @@ exports[`CSS snapshots variables in medicare-theme.css match snapshot 1`] = `
...
 --label__font-weight: 600;
 --label__letter-spacing: 0.5px;
 --label__text-transform: uppercase;
---lead-max-width: 77rem;
---link-inverse__color--active: #d9d9d9;
---link-inverse__color--hover: #d9d9d9;
---link-inverse__color--visited: #d9d9d9;
+--link-inverse__color--active: #e9ecf1;
+--link-inverse__color--hover: #e9ecf1;
+--link-inverse__color--visited: #e9ecf1;
```
Comment (pwolfert) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:1832]: When these moved to Figma, we noticed the difference between this theme and the others and decided to make it conform. It previously used `gray-lighter`, and now it uses `muted-inverse`. CC @hi-its-hailey

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap (hunk 1 above)
Comment (zarahzachz) [reply packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:31]: are the `--animation-speed-*` vars used? i think i'm only seeing speed 2

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap [hunk 7]
```diff
@@ -367,7 +389,6 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
 --media-width-sm: 544px;
```
Comment (zarahzachz) [inline packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:389]: can these `--media-width-*` vars be used? seems like they'd only appear in media queries but you can't use css vars in media queries

**Review by zarahzachz — APPROVED** [review]

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap (hunk 1 above)
Comment (pwolfert) [reply packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:31]: I'm seeing 1, 2, and 4 being used

File: packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap (hunk 7 above)
Comment (pwolfert) [reply packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap:389]: You're right. They might not be super useful, but at least users have the values to reference? I don't know.

**Pull request merged**